app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -144,8 +144,7 @@ class ApplicationModule(context: Context)
       controllerComponents
     )
 
-  val graphiteModule =
-    new GraphiteModule(environment, configuration, configModule.runMode, applicationLifecycle, metricsModule)
+  new GraphiteModule(environment, configuration, configModule.runMode, applicationLifecycle, metricsModule)
 
```

Comment:
as this is not assigned to any val, do we need this?

Reply:
Good question. I think we need it due to side effects.

Reply:
Can we add a comment please?